development/site-components/navbar.pug

```diff
@@ -0,0 +1,26 @@
+html
```

Comment:
please rename folder **includes* this is considered more standard in the industry and probably more accurate given some items in here will be included but are not components rather than just files, or meta info.